student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative './person'
+
+class Student < Person
+  def initialize(classroom)
+    super()
+    @classroom = classroom
+  end
```

Comment:
- [ ] the constructor should extend the parent's constructor by adding @classroom and a parameter for it. have a look at [this example](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_principles_examples.md#inheritance)

Reply:
@mikethreels which means? I apologize but still don't get it.

Reply:
- [ ] the constructor should look something like the below example, otherwise, the variables aren't passed to the parent correctly:
```
  def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @classroom = classroom
  end
```

Reply:
Thanks Changes made ;)